### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by mrspence** [Gustash/react-native-smart-lock#9]
Is this a dead project?

- Old React Version
- Gradle 8 not supported
- Last Git Release not published on NPM

_Author(s): thank you for the work you have put into this, maintaining open source projects is hard work! Putting this issue here as a warning for other developers, in case this **is** a dead project now._

**Comment by habovh** [comment]
It very much seems like it. I don't think updating the library would require too much work, however it definitely requires a deeper knowledge and understanding of the Android build process than I do.

For now, I resigned to remove the Smart Lock features altogether because I desperately need to update to a more recent React Native version that also requires Gradle 7 —which seems not to be too happy with this lib either.

Maybe an Android specialist could have a look at the `build.gradle` and assess? Would love to keep this package working!

**Comment by dylanhillier** [comment]
Yeah, this module shouldn't be used anymore because SmartLock is considered deprecated from what i understand:
https://developers.google.com/identity/one-tap/android/overview

Its also worth pointing out that the built in android autofill can be used instead.

Failing that, you can probably look at something like https://github.com/Spicy-Sparks/react-native-google-one-tap-signin as an alternative. See the `password_credential_request_support` branch of my fork as an example.
